src/gui/animscript.cpp

```diff
@@ -27,11 +28,7 @@ AnimScript::~AnimScript(){
 }
 
 QString AnimScript::path(){
-#ifdef __APPLE__
-    return QDir(QApplication::applicationDirPath() + "/../Resources").absoluteFilePath("ckb-animations");
-#else
-    return QDir("/usr/lib").absoluteFilePath("ckb-animations");
-#endif
+    return QString(CKB_NEXT_ANIMATIONS_PATH);
```

Comment:
Not strictly related to this, but maybe in the future we should return a QStringList or something along those lines, so that we can have the possibility of both user-defined and local-relative paths for animations.

Reply:
Ok, leaving it to be done afterwards, after merge.